cypress/integration/parallel-2/productSwitch.spec.ts

```diff
@@ -270,6 +297,8 @@ describe('product switching', () => {
 			cy.findByText(
 				/Your first billing date is today and you will be charged £5/,
 			).should('exist');
+
+			cy.get('@mdapi_get_contribution.all').should('have.length', 1);
```

Comment:
Is it worth adding a test where there is more than one?

Reply:
I actually meant to do this, but put it in the wrong setup